template/codeland.rb

```diff
@@ -197,11 +203,21 @@ def user_not_authorized
     end
   end
 
+  empty_directory 'decorators' do
+    create_file '.keep'
+  end
+
   inside 'views' do
     directory 'partials'
   end
 end
 
+inside 'spec' do
+  empty_directory 'decorators' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.